src/status_im2/common/toasts/view.cljs

```diff
@@ -0,0 +1,98 @@
+(ns status-im2.common.toasts.view
+  (:require
+   [quo2.components.notifications.toast :as quo2.toast]
+   [react-native.core       :as rn]
+   [react-native.gesture    :as gesture]
+   [react-native.reanimated :as reanimated]
+   [reagent.core            :as reagent]
+   [status-im.utils.utils   :as utils.utils]
+   [status-im2.common.toasts.subs]
+   [utils.re-frame          :as re-frame]))
+
+(def ^:private slide-out-up-animation
+  (-> reanimated/slide-out-up-animation
+      .springify
+      (.damping 20)))
+
+(def ^:private slide-in-up-animation
+  (-> reanimated/slide-in-up-animation
+      .springify
+      (.damping 20)))
+
+(def ^:private linear-transition
+  (-> reanimated/linear-transition
+      .springify
+      (.damping 20)))
+
+(defn container
+  [id]
+  (let [toast-opts         (re-frame/sub [:toasts/toast id])
+        duration           (get toast-opts :duration 3000)
+        on-dismissed       (get toast-opts :on-dismissed identity)
+        dismissed-locally? (reagent/atom false)
+        close!             #(re-frame/dispatch [:toasts/close id])
+        new-timer          #(utils.utils/set-timeout close! duration)
+        timer              (reagent/atom (new-timer))
+        clear-timer        #(utils.utils/clear-timeout @timer)]
+    [:f>
+     (fn []
+       (rn/use-unmount #(do (clear-timer) (on-dismissed id)))
+       (let [translate-y (reanimated/use-shared-value 0)
+             pan
+             (->
+              (gesture/gesture-pan)
+              ;; remove timer on pan start
+              (gesture/on-start clear-timer)
+              (gesture/on-update
+               (fn [evt]
+                 (let [evt-translation-y (.-translationY evt)]
+                   (cond
+                     ;; reset translate y on pan down
+                     (> evt-translation-y 100)
+                     (reanimated/animate-shared-value-with-spring translate-y
+                                                                  0
+                                                                  {:mass    1
+                                                                   :damping 20
+                                                                   :stiffness
+                                                                   100})
```

Comment:
Thank you for the animation files. Please can you take a look at this, stiffness value is different from the design values

![image](https://user-images.githubusercontent.com/17097240/205669752-7bacb00a-d8d7-4759-bbed-62abdc907434.png)

Reply:
done